src/models/wallet.ts

```diff
@@ -21,38 +21,38 @@ class Wallet {
         return this.generateLink(paymentRequest)
     }
 
-    static wallets = [
-        new Wallet(
+    static wallets = {
+        lightning: new Wallet(
```

Comment:
The order of the provider wallets should be alphabetical, but we can put lightning first